scripts/release/constants.mjs

```diff
@@ -15,3 +15,9 @@ export const [OWNER, REPO] = process.env.GITHUB_REPOSITORY.split("/");
 export const LATEST_RELEASE = process.env.GITHUB_REF.replace("refs/tags/", "");
 export const GITHUB_TOKEN = process.env.GITHUB_TOKEN;
 export const GITHUB_REPOSITORY = process.env.GITHUB_REPOSITORY;
+
+/**
+  if you're changing the paths to be counted,
+  make sure you also update /.github/workflows/release.yml
+ */
```

Comment:
Is this still correct?

Reply:
nope, maybe one day when i figure out how to get a proper commit diff as `$(git rev-list $GITHUB_SHA..HEAD --count)` results in 0 as $GITHUB_SHA and HEAD are the same commit, I have ideas, but for now this will work, we just may have a few days were unneeded nightlies are released